lib/harvester.js

```diff
@@ -433,6 +453,7 @@ Harvester.prototype._removeRoutes = function (name, methods, routes) {
 
   this.adapter.awaitConnection().then(function () {
     (methods || []).forEach(function (verb) {
+//        TODO imho this method is badly implemented. `router` does not have `routes` property.
```

Comment:
I dont think this works; express does not have an api for deregistering routes, so what you're seeing is a hack that may have worked for an earlier version of express.